module/actor/roll-dice.js

```diff
@@ -0,0 +1,110 @@
+
+/* global ChatMessage, Roll, game */

```

Comment:
The roll dice function is cool, but I thought  we should put it on its own place :)